addons/l10n_ua/data/template/account.tax-ua_ias.csv

```diff
@@ -1,81 +1,74 @@
-"","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund","ua_ias_1140"
-"purchase_tax_template_vat7","21","Придбання з ПДВ 7%","","+ ПДВ 7%","7.0","purchase","tax_group_vat7","","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_ias_1140"
-"","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund","ua_ias_1140"
-"purchase_tax_template_vat7incl","21","Придбання в т. ч. ПДВ 7%","","в т. ч. ПДВ 7%","7.0","purchase","tax_group_vat7","True","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_ias_1140"
-"","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund","ua_ias_1140"
-"purchase_tax_template_vat0","22","Придбання з ПДВ 0%","","ПДВ 0%","0.0","purchase","","","base","invoice",""
-"","","","","","","","","","tax","invoice",""
-"","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund",""
-"purchase_tax_template_vat_free","23","Придбання звільнене від ПДВ","","Звільнено від ПДВ","0.0","purchase","","","base","invoice",""
-"","","","","","","","","","tax","invoice",""
-"","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund",""
-"purchase_tax_template_vat_not","24","Придбання не є об'єктом ПДВ","","Не є об'єктом ПДВ","0.0","purchase","","","base","invoice",""
-"","","","","","","","","","tax","invoice",""
-"","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund",""
-"simple_tax_sale_product","30","Дохід від продажу товарів","","Дохід від продажу товарів","0.0","sale","","","","",""
-"simple_tax_sale_gift","31","Дохід від безоплатно отриманих товарів","","Дохід від безоплатно отриманих товарів","0.0","sale","","","","",""
-"simple_tax_sale_old","32","Дохід від заборгованності, за якою минув строк позивної давності","","Дохід від заборгованності, за якою минув строк позивної давності","0.0","sale","","","","",""
-"simple_tax_sale_15","33","Дохід, що оподатковується за ставкою 15%","","Дохід за ставкою 15%","0.0","sale","","","","",""
-"simple_tax_purchase_product","40","Витрати від продажу товарів","","Витрати від продажу товарів","0.0","purchase","","","","",""
-"simple_tax_purchase_salary","41","Витрати на оплату праці найманих працівників","","Витрати на оплату праці найманих працівників","0.0","purchase","","","","",""
-"simple_tax_purchase_esv","42","Витрати на ЄСВ","","Витрати на ЄСВ","0.0","purchase","","","","",""
-"simple_tax_purchase_other","43","Витрати інші","","Витрати інші","0.0","purchase","","","","",""
+"id","sequence","name","description","invoice_label","amount","type_tax_use","tax_group_id","active","price_include","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","name@uk_UA","invoice_label@uk_UA"
+"sale_tax_template_vat20","9","20%","","20%","20.0","sale","tax_group_vat20","","","base","invoice","","Реалізація з ПДВ 20%","+ ПДВ 20%"
```

Comment:
Just to be sure the invoice label is completely changed compared to the original one, do we want that ?